test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/ActivityDetailsViewSpec.scala

```diff
@@ -322,22 +324,40 @@ class ActivityDetailsViewSpec extends ViewSpec {
 
       // Then
       doc should containElementWithID("activity-events-assignee")
-      doc.getElementById("activity-events-assignee") should containText("PID id")
+      doc.getElementById("activity-events-assignee").text() shouldBe "PID id"
+      doc.getElementById("activity-events-assignee-label").text() shouldBe "Currently assigned to:"
     }
 
-    "Render 'Unassigned'" in {
+    "Render currently in 'ACT'" in {
       // Given
       val c = aCase(
-        withAssignee(None)
+        withAssignee(None),
+        withQueue(Queues.act.id)
       )
 
       // When
       val doc = view(activity_details(c, Paged.empty[Event], ActivityForm.form))
 
       // Then
-      doc should containElementWithID("activity-events-assignee")
-      doc.getElementById("activity-events-assignee") should containText("unassigned")
+      doc should containElementWithID("activity-events-assigned-queue")
+      doc.getElementById("activity-events-assigned-queue").text() shouldBe "ACT"
+      doc.getElementById("activity-events-assigned-queue-label").text() shouldBe "Currently in:"
     }
   }
 
+    "Render currently in 'Gateway'" in {
+      // Given
+      val c = aCase(
+        withAssignee(None)
+      )
+
+      // When
+      val doc = view(activity_details(c, Paged.empty[Event], ActivityForm.form))
+
+      // Then
+      doc should containElementWithID("activity-events-assigned-queue")
+      doc.getElementById("activity-events-assigned-queue").text() shouldBe "Gateway"
+      doc.getElementById("activity-events-assigned-queue-label").text() shouldBe "Currently in:"
```

Comment:
I'm not sure we need the `activity-events-assigned-queue-label` assertion as theres nothing dynamic in there